spectrum-network/src/peer_manager.rs

```diff
@@ -240,6 +240,10 @@ pub struct NetworkingConfig {
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub struct PeerManagerConfig {
+    /// A peer with reputation below this value is classed as 'misbehaving', and its connection
+    /// should be dropped.
+    pub misbehaving_peer_threshold: Reputation,
+    /// Represents the minimum reputation a peer must have to accept its incoming connection.
     pub min_reputation: Reputation,
```

Comment:
I think we should keep the naming of these two settings consistent (e.g `min_drop_reputation`, `min_accept_reputation` or something like that)

Reply:
Sure, will use `min_acceptable_reputation`.